spec/controllers/spree/admin/general_settings_controller_decorator_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe Spree::Admin::GeneralSettingsController, type: :controller do
+  let(:user) { create(:user) }
+
+  before do
+    allow(controller).to receive_messages spree_current_user: user
+    user.spree_roles << Spree::Role.find_or_create_by(name: 'admin')
+  end
+
+  def do_update
+    post :update, disable_quote?: true, quotes_count: 10, store: { url: 'www.spree.com' }
```

Comment:
still disable_quote`?`, have you run the tests before submitting? is it working?